evil-visual-mark-mode.el

```diff
@@ -52,8 +52,7 @@
 
 This marker will normally come from the advised evil-set-marker
 function."
-  (let* ((marker-pos (marker-position marker)))
-    (make-overlay marker-pos (+ 2 marker-pos))))
+  (make-overlay marker marker))
```

Comment:
I'm not familiar with how the overlay mechanism works underhood, perhaps I misunderstand something. 
The official Emacs Lisp reference says the `before-string` property won't appear in the buffer in any sense except for visual effects, so I think it doesn't really occupy any space that denoted by `buffer position`. In my Emacs, it seems that the visual markers are rendered correctly even with zero-length overlays.